docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -21,24 +21,37 @@ aliases:
   - /installation/viam-micro-server-dev/
 ---
 
-`viam-micro-server` is the lightweight version of [`viam-server`](/operate/reference/viam-server/) which can run on resource-limited embedded systems (ESP32) that cannot run the fully-featured `viam-server`.
-
-This page provides instructions for configuring a development environment for working with `viam-micro-server`, outlines the steps for performing various development tasks, and provides troubleshooting and development tips to help organize and streamline work.
+[The Micro-RDK](https://github.com/viamrobotics/micro-rdk/) is the lightweight version of Viam's full Robot Development Kit, designed for resource-limited embedded systems (ESP32) that cannot run the fully-featured `viam-server`.
+`viam-micro-server` is the pre-built firmware built from the Micro-RDK and a default set of {{< glossary_tooltip term_id="module" text="modules" >}}, provided for convenience.
+If you want to use different modules with the Micro-RDK, you can build your own firmware using the instructions on this page.
 
 {{< alert title="Looking to install viam-micro-server?" color="note" >}}
 If you only want to install the pre-built `viam-micro-server` firmware with a default set of modules, follow the normal [setup instructions](/operate/get-started/setup/) instead.
 {{< /alert >}}
 
+{{< expand "Why does the Micro-RDK work differently from the full RDK?" >}}
+Microcontrollers do not have full operating systems like single-board computers and general-purpose computers.
+This means that it is generally harder to make software portable across multiple microcontrollers, and it means that microcontrollers can only run one "program" at a time.
+
+Rather than being able to download and install an executable the way you can on a computer or single-board computer, a microcontroller must be flashed with firmware containing the entire logic to run.
```

Comment:
```suggestion
This means that microcontrollers can only run one "program" at a time, and a microcontroller must be flashed with firmware containing the entire logic to run.
```

Suggesting to remove the part about portability; that's more an internal note on why we have been carefully selecting which microcontroller models to support :)

Reply:
Ah okay great thank you, makes sense--and I'm happy to cut out excess text anyway.